rust/processor/src/db/postgres/migrations/2024-10-07-123511_schema_autoreload/up.sql

```diff
@@ -0,0 +1,63 @@
+-- Your SQL goes here
```

Comment:
```suggestion
-- This is taken directly from the PostgREST schema cache docs.
```

Reply:
You can leave a link to this here too, I think it would appropriate: https://docs.postgrest.org/en/v12/references/schema_cache.html